<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IRC</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_send">
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_Username">
           <property name="text">
            <string>Username:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Username"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Connect">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit_Username</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton_Connect</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton_send</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
